service/src/main/java/bio/terra/pipelines/app/configuration/internal/PipelinesCommonConfiguration.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.pipelines.app.configuration.internal;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "pipelines.common")
+public record PipelinesCommonConfiguration(Long storageBucketTtlDays) {}
```

Comment:
any reason for this to be a Long vs an Integer?

Reply:
its onlly cuz the `Instant.add()` function takes in a Long value and not an int